azkaban-common/src/test/java/azkaban/executor/SelectorTest.java

```diff
@@ -602,4 +605,101 @@ public void testSelectorListWithNullValue() throws Exception {
     Assert.assertTrue(null == executor);
 
   }
+
+  @Test
+  public void testCreatingExectorfilterObject() throws Exception{
+    List<String> validList = new ArrayList<String>(ExecutorFilter.getAvailableFilterNames());
+    try {
+      new ExecutorFilter(validList);
+    }catch (Exception ex){
+      Assert.fail("creating ExecutorFilter with valid list throws exception . ex -" + ex.getMessage());
+    }
+  }
+
+  @Test
```

Comment:
we may need to upgrade junit version, I tried it some time back but there were some issues